src/github_changelog/cli.clj

```diff
@@ -42,13 +45,13 @@
 
 (defn -main [& args]
   (let [{:keys [options arguments errors summary]} (parse-opts args cli-options)
-        {:keys [help last]}                        options]
+        {:keys [help last since]}                  options]
     (cond
       help               (exit 0 (usage summary))
       (empty? arguments) (exit 1 (usage summary))
       errors             (exit 1 (error-msg errors)))
 
     (doseq [config-file arguments]
-      (println (generate config-file last)))
+      (println (generate config-file last since)))
```

Comment:
I would rather pass the whole options and destruct the necessary key in the `generate` function